fortinet-fortigate/package.yaml

```diff
@@ -0,0 +1,38 @@
+id: fortinet.fortigate
+name: Fortinet FortiGate
+author: Tenzir
+author_icon: https://raw.githubusercontent.com/tenzir/library/main/author.svg
+package_icon: https://raw.githubusercontent.com/tenzir/library/main/fortinet-fortigate/package.svg
+
+description: |
+  [Fortinet FortiGate] (https://www.fortinet.com/products/next-generation-firewall) appliances provide network security by combining firewall, VPN, and intrusion prevention features.
+
+  This package makes it easy to onboard the logs that FortiGate generates.
+
+inputs:
+  syslog-ip-adress:
+    name: IP adress
+    description: A UDP endpoint where FortiGate sends its syslogs.
+    type: string
+    default: 127.0.0.1
+  syslog-port:
+    name: Syslog Port
+    description: An UDP endpoint for receiving FortiGate logs via syslog.
+    type: string
+    default: 514
+
+pipelines:
+ onboard-from-udp:
+    name: Onboard FortiGate logs via UDP
+    description: A pipeline for ingesting and processing FortiGate logs received via UDP.
+    definition: |
+      from "udp:// {{ inputs.syslog-ip-adress }}:{{ inputs.syslog-port }}" read syslog
+      | parse content kv
+      | yield content
+      | set #schema = "fortinet.fortigate"
+      | publish "fortinet"
+
+examples:
+
+       
+ 
```

Comment:
```suggestion
```

It's okay not to have examples for this one for now, I think. We need more experience with the data first.